source/packaging-guide/optional/appstream.rst

```diff
@@ -3,13 +3,13 @@
 AppStream metadata
 ==================
 
-AppStream is a cross-distro effort for providing metadata for software in the (Linux) ecosystem.
+AppStream is a cross-distribution effort for providing metadata for software in the (Linux) ecosystem.
```

Comment:
`distro` isn't wrong per se, but the long version is probably easier to understand. Good catch.